pkg/api/router/router.go

```diff
@@ -34,7 +35,16 @@ func healthz(c *gin.Context) {
 	return
 }
 
+func mountMetrics(r *gin.Engine) {
+	r.GET("/metrics", metrics)
```

Comment:
Will it conflict with the user's existing API?

Reply:
It is the API of apisix ingress controller itself, users cannot add apis to it.

Reply:
got it